config/initializers/authenticator.rb

```diff
@@ -6,11 +6,6 @@
 
 Sequel.extension :migration
 
-# Token authentication is optional for authn routes, and it's not applied at all to authentication.
-Possum::Application.config.middleware.use Conjur::Rack::Authenticator,
```

Comment:
Just curious why we moved this?

Reply:
When I was figuring out how to add the middleware I noticed this; I figured it makes more sense to have it in a configuration file than in an initializer which ought to be more generic. I don't feel strongly on this subject however :)